### PR TITLE
Themed css

### PR DESCRIPTION
**Pull request by jeffbryner**

### CODE REVIEW

**Comment by Phrozyn** [comment]
Merged master  changes into this branch to resolve conflicts.

**Comment by pwnbus** [comment]
@jeffbryner When I tried to build the meteor container from this branch, I'm getting an error now:
```
Errors prevented bundling:
While processing files with templating-compiler (for target web.browser):
client/menu.html:40: Unexpected HTML close tag
...                </ul>                    ...
^

While processing files with templating-compiler (for target
web.browser.legacy):
client/menu.html:40: Unexpected HTML close tag
...                </ul>                    ...
```

**Comment by Phrozyn** [comment]
@jeffbryner I was wondering about adding a link to the documentation as a sub-menu item under About. Do you have any feedback, for or against?

**Comment by jeffbryner** [comment]
> @jeffbryner I was wondering about adding a link to the documentation as a sub-menu item under About. Do you have any feedback, for or against?

Sure, link to readthedocs?

**Comment by Phrozyn** [comment]
> > @jeffbryner I was wondering about adding a link to the documentation as a sub-menu item under About. Do you have any feedback, for or against?
> 
> Sure, link to readthedocs?

Yes :)

**Comment by Phrozyn** [comment]
> @jeffbryner When I tried to build the meteor container from this branch, I'm getting an error now:
> 
> ```
> Errors prevented bundling:
> While processing files with templating-compiler (for target web.browser):
> client/menu.html:40: Unexpected HTML close tag
> ...                </ul>                    ...
> ^
> 
> While processing files with templating-compiler (for target
> web.browser.legacy):
> client/menu.html:40: Unexpected HTML close tag
> ...                </ul>                    ...
> ```

This is still an issue as I try to build it for testing. I've reviewed the code and spent some time combing through it but can't tell where the offending issue is that is causing it.

Is the docker version with these changes deploying perfectly for you?

**Comment by jeffbryner** [comment]
> > @jeffbryner When I tried to build the meteor container from this branch, I'm getting an error now:
> > ```
> > Errors prevented bundling:
> > While processing files with templating-compiler (for target web.browser):
> > client/menu.html:40: Unexpected HTML close tag
> > ...                </ul>                    ...
> > ^
> > 
> > While processing files with templating-compiler (for target
> > web.browser.legacy):
> > client/menu.html:40: Unexpected HTML close tag
> > ...                </ul>                    ...
> > ```
> 
> This is still an issue as I try to build it for testing. I've reviewed the code and spent some time combing through it but can't tell where the offending issue is that is causing it.
> 
> Is the docker version with these changes deploying perfectly for you?

Just found the issue, the "if" statement for the watchlist was missing a { in the /if. Should be fixed now.

**Comment by Phrozyn** [comment]
I just thought about this, you might want to capture my code in https://github.com/mozilla/MozDef/pull/1207 to fix that whiteout issue that happens when you hover across investigations/incidents.

File: meteor/client/investigationEdit.html [hunk 1]
```diff
@@ -7,536 +7,585 @@
...
-                <div class="form-group">
-                    <label class="col-xs-2 control-label" for="summary">Summary:
-                        <i class="fa fa-info-circle" data-toggle="tooltip" title="Enter 'Category': 'Reason for Investigation' on 'Affected Target(s)' "></i>
-                    </label>
-                    <div class="col-xs-10">
-                        <input id="summary" name="summary" placeholder="Detailed description of the investigation" class="summary form-control" required type="text" value="{{summary}}">
-                    </div>
-                </div>
+    <div class="container">
+        <div id="maintab" class="tabcontent active ">
```
[the remark this answers is not in the document]
Comment (jeffbryner) [reply meteor/client/investigationEdit.html]: cool, fixed.

File: meteor/client/incidentEdit.html [hunk 2]
```diff
@@ -7,451 +7,498 @@
...
-            <li class="tabnav active"><a href="#maintab">Main</a></li>
-            <li class="tabnav"><a href="#notestab">Notes</a></li>
-            <li class="tabnav"><a href="#referencestab">References</a></li>
-            <li class="tabnav"><a href="#theoriestab">Theories</a></li>
-            <li class="tabnav"><a href="#timestampstab">Timestamps</a></li>
-            <li class="tabnav"><a href="#mitigationstab">Mitigations</a></li>
-            <li class="tabnav"><a href="#lessonstab">Lessons Learned</a></li>
-        </ul>
+    <div class="container">
+        <div class="row col-xs-9 center ">
```
[the remark this answers is not in the document]
Comment (jeffbryner) [reply meteor/client/incidentEdit.html]: fixed it up

**Comment by jeffbryner** [comment]
> I just thought about this, you might want to capture my code in #1207 to fix that whiteout issue that happens when you hover across investigations/incidents.

OK, took a shot at it. It's better. I think we can fix it permanently after this PR lands by revisiting: https://github.com/mozilla/MozDef/blob/master/meteor/client/investigations.js#L47 which causes this by setting a tooltip.

**Comment by Phrozyn** [comment]
> > I just thought about this, you might want to capture my code in #1207 to fix that whiteout issue that happens when you hover across investigations/incidents.
> 
> OK, took a shot at it. It's better. I think we can fix it permanently after this PR lands by revisiting: https://github.com/mozilla/MozDef/blob/master/meteor/client/investigations.js#L47 which causes this by setting a tooltip.

sounds good :)

**Pull request merged**